govc/dvs/create.go

```diff
@@ -94,6 +99,18 @@ func (cmd *create) Run(ctx context.Context, f *flag.FlagSet) error {
 		}
 	}
 
+	numUplinkPorts := int(cmd.numUplinkPorts)
+
+	if numUplinkPorts > 0 {
+		var policy types.DVSNameArrayUplinkPortPolicy
+		for i := 0; i < numUplinkPorts; i++ {
+			policy.UplinkPortName = append(policy.UplinkPortName, fmt.Sprintf("Uplink %d", i+1))
+		}
+		cmd.configSpec.UplinkPortPolicy = &policy
+	} else {
+		return errors.New("Must specify more than one uplink port")
```

Comment:
This isn't required to create a DVS though, is it?  Seems like we can keep the default as `0`, without erroring here.

Reply:
I was trying to have this cover if someone put in a negative number or hardcoded zero.  Originally I wasn't sure if there was an upper limit on the number of uplinks, but there doesn't appear to be.  Happy to remove if not appropriate.

Reply:
Looks like if you use `f.UintVar(...)`, then the Go flag package will complain about a negative value, and `0` value will just result in vSphere's default behavior as the command does currently.